Argo/Types/StandardTypes.swift

```diff
@@ -72,11 +157,46 @@ public extension CollectionType where Generator.Element: Decodable, Generator.El
   }
 }
 
+/**
+  Decode `JSON` into an array of values where the elements of the array are
+  `Decodable`.
+
+  If the `JSON` is an array of `JSON` objects, this returns a decoded array
+  of values by mapping the element's `decode` function over the `JSON` and
```

Comment:
Is that right? We're applying the `decode` function for the specific element. Not the decode function for all of the elements.